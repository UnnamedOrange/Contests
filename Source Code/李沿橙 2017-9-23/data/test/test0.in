5
4
1 2 3 4 
3
1000 500 1 
5
1 3 4 5 7 
50
1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 
50
1000 456 65 24 62 523 45 234 653 453 523 51 5 134 13 453 45 354 134 312 523 923 452 345 845 234 523 523 45 235 234 523 452 345 9 75 4 345 31 123 54 5 54 54 543 25 45 4 999 1000 
