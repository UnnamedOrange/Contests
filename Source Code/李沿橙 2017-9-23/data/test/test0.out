1 2 4 3 
1 500 1000 
1 3 5 7 4 
1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 
4 4 5 5 9 13 24 25 31 45 45 45 45 51 54 54 54 62 65 75 123 134 134 234 234 234 312 345 345 345 354 452 452 453 453 456 523 523 523 523 523 543 845 999 1000 1000 923 653 523 235 
